tests/test_plot.py

```diff
@@ -10,7 +10,7 @@
 from pyfar import Signal
 
 # flag for creating new baseline plots (required if the plot look changed)
-create_baseline = False
+create_baseline = True
```

Comment:
Reset to false?

Reply:
Yes, thx for noting - it's time to use enviornment variables for this :)